server/remote/gitea/helper.go

```diff
@@ -176,23 +176,40 @@ func buildFromPullRequest(hook *pullRequestHook) *model.Build {
 	return build
 }
 
-// helper function that extracts the Repository data from a Gitea push hook
-func repoFromPush(hook *pushHook) *model.Repo {
-	return &model.Repo{
-		Name:     hook.Repo.Name,
-		Owner:    hook.Repo.Owner.Username,
-		FullName: hook.Repo.FullName,
-		Link:     hook.Repo.URL,
+func buildFromRelease(hook *releaseHook) *model.Build {
+	avatar := expandAvatar(
+		hook.Repo.URL,
+		fixMalformedAvatar(hook.Sender.Avatar),
+	)
+	author := hook.Sender.Login
+	if author == "" {
+		author = hook.Sender.Username
+	}
+	sender := hook.Sender.Username
+	if sender == "" {
+		sender = hook.Sender.Login
+	}
+
+	return &model.Build{
+		Event:     model.EventRelease,
+		Ref:       fmt.Sprintf("refs/tags/%s", hook.Release.TagName),
+		Link:      fmt.Sprintf("%s/src/tag/%s", hook.Repo.URL, hook.Release.TagName),
+		Branch:    fmt.Sprintf("refs/tags/%s", hook.Release.TagName),
+		Message:   fmt.Sprintf("created release %s", hook.Release.TagName), <- release Title?
```

Comment:
```suggestion
		Message:   fmt.Sprintf("created release %s", hook.Release.Name),
```

Reply:
... I think we can use that :)

Reply:
Yea that works as well. I think both would work since we exepect the release to be the tag. At least for github.